store/sql_user_store.go

```diff
@@ -50,6 +50,7 @@ func NewSqlUserStore(sqlStore *SqlStore) UserStore {
 		table.ColMap("NotifyProps").SetMaxSize(2000)
 		table.ColMap("Locale").SetMaxSize(5)
 		table.ColMap("MfaSecret").SetMaxSize(128)
+		table.ColMap("Position").SetMaxSize(35)
```

Comment:
I would set it to something more std like 64 chars then limit in the UI or via checks.  We do this all over the place because we change our minds so often on size.  That way the DB doesn't need to change, just a random constant somewhere.